kotest-assertions/kotest-assertions-core/src/jvmTest/kotlin/com/sksamuel/kotest/matchers/collections/CollectionMatchersTest.kt

```diff
@@ -260,21 +257,6 @@ class CollectionMatchersTest : WordSpec() {
          }
       }
 
-      "haveDuplicates" should {
-         "test that a collection is unique" {
-            listOf(1, 2, 3, 3) should containDuplicates()
-            listOf(1, 2, 3, 4) shouldNot containDuplicates()
-            listOf(1, 2, 3, 3).shouldContainDuplicates()
-            listOf(1, 2, 3, 4).shouldNotContainDuplicates()
-         }
-
-         "print duplicates in message" {
-            shouldThrowAny {
-               listOf(1, 2, 3, 4, 2, 1) shouldNot containDuplicates()
-            }.shouldHaveMessage("Collection should not contain duplicates, but has some: [1, 2]")
```

Comment:
These tests were removed because they are duplicates (See DuplicatesTest.kt)

Reply:
Duplication tests were themselves duplicated. I don't know why that's so funny to me :D